src/api/SFA.DAS.Provider.Events.Api.IntegrationTests/Setup/DatabaseSetup.cs

```diff
@@ -36,6 +39,9 @@ public DatabaseSetup(PopulateTables populate)
         {
             Debug.WriteLine("Populating submission events table, this could take a while");
 
+            await PopulateEventsForByUln(1000000160, 2);
```

Comment:
Does this databaset setup method get called for each test?  If it does, we don't want to insert the ULN events each time if most tests don't need them.  If it's a one off call for each assembly then it's fine.

Reply:
This is only called once at the start.